### DISCUSSION TITLE
Dimmed class overlay is applied to adjacent other HTML element

### DISCUSSION THREAD

**Discussion by charlycou** [quasarframework/quasar#14083] [General - Components / Directives / etc] [answered]
I'm trying to use the "[dimmed](https://quasar.dev/style/visibility#introduction)" class but I realize that the dimmed overlay is applied to all adjacent elements of the element where the class is set.

https://codepen.io/charlycou/pen/OJvOgLj?editors=111

Anyone already manage to apply the dimmed on one particular HTML element?

**Comment by pdanpdan** [comment, marked as answer]
You must use it on a relative/absolute/fixed positioned element
https://codepen.io/pdanpdan/pen/dymZdeb?editors=111
